src/gt4py/next/program_processors/runners/dace_fieldview/gtir_dataflow.py

```diff
@@ -1309,3 +1320,38 @@ def visit_SymRef(self, node: gtir.SymRef) -> IteratorExpr | MemletExpr | SymbolE
         # if not in the lambda symbol map, this must be a symref to a builtin function
         assert param in gtir_python_codegen.MATH_BUILTINS_MAPPING
         return SymbolExpr(param, dace.string)
+
+    def apply(
```

Comment:
I don't remember that there is a call to `apply()`.
As far as I can tell `apply` is only called in this function, so it is a recursive function.
However, it does some highly non trivial pre and post processing.
So I think that:
- This function needs a better name, `apply` is just too generic.
- This function needs a doc string.
- This function needs comment that explain why the pre and post processing is needed and how it is done.

Reply:
I would like to keep the name `apply` for the entry point of this visitor class, since it is consistent with other visitor classes in GT4Py. However, I agree on the rest so I will write some documentation.

Reply:
I still think that `apply()` is the wrong name. If it is for compatibility, then why was it there before?
`_vistit_let()` should be the much better name.

Reply:
I have now made a code change in this direction.